.idea/jarRepositories.xml

```diff
@@ -9,17 +9,37 @@
     <remote-repository>
       <option name="id" value="local-maven-repo" />
       <option name="name" value="local-maven-repo" />
-      <option name="url" value="file:////local-maven-repo" />
+      <option name="url" value="file:////home/paulmicu/Projects/notorious-eng/JavaFXUserInterface/local-maven-repo" />
```

Comment:
Has this been causing an issue?
All your paths are specific to your machine, they should be relative.

Reply:
Looks like an .idea file was committed accidentally. I'll remove it from the coming